hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/OperationSubsystemStart.java

```diff
@@ -44,7 +44,17 @@ public void execute(OperationContext opContext, ModelNode model) throws Operatio
                 LOGGER.debug("Asked to start the Hawkular Monitor service");
             }
             if (!service.isMonitorServiceStarted()) {
-                service.startMonitorService();
+                Thread newThread = new Thread(new Runnable() {
+                    @Override
+                    public void run() {
+                        try {
+                            service.startMonitorService();
+                        } catch (Throwable t) {
+                        }

```

Comment:
But those Throwables are already catched by `startMonitorService` and as far as i can see they aren't re-throw.
There wasn't any try/catch surrounding `service.startMonitorService()` before, why do we need one now?